image_provisioner/openshift-templates/pbench/pbench-agent-daemonset.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: extensions/v1beta1
```

Comment:
openshift-templates directory is not just for a particular role, we might add some other stuff which might not be part of pbench-kickstart role. So I think it's better to keep this as a common directory to store templates though we can place it in the files directory in the role in this case.